cmd/util/ledger/migrations/account_migration.go

```diff
@@ -0,0 +1,94 @@
+package migrations
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/fvm/environment"
+	"github.com/onflow/flow-go/fvm/meter"
+	"github.com/onflow/flow-go/fvm/state"
+	"github.com/onflow/flow-go/ledger"
+)
+
+func MigrateAccountUsage(payloads []ledger.Payload, pathFinderVersion uint8) ([]ledger.Payload, []ledger.Path, error) {
+	return MigrateByAccount(AccountUsageMigrator, payloads, pathFinderVersion)
+}
+
+func KeyToStorageInteractionKey(key ledger.Key) (meter.StorageInteractionKey, error) {
+	id, err := KeyToRegisterID(key)
+	if err != nil {
+		return meter.StorageInteractionKey{}, err
+	}
+	return meter.StorageInteractionKey{
+		Owner: id.Owner,
+		Key:   id.Key,
+	}, nil
+}
+
+func payloadSize(key ledger.Key, payload ledger.Payload) (uint64, error) {
+	ik, err := KeyToStorageInteractionKey(key)
+	if err != nil {
+		return 0, err
+	}
+
+	return meter.GetStorageKeyValueSizeForTesting(ik, payload.Value()), nil
```

Comment:
In order to get the cached storage size for the payload, I need to call this function which is a testing function. Would it make sense to remove the `Testing` in the name, and make it public? @pattyshack .

Also , is this the right function to compute the storage size of a given payload? @janezpodhostnik